resources/ts/Shared/SecondaryButton.tsx

```diff
@@ -1,18 +1,24 @@
-import React from "react";
-import { Link, InertiaLinkProps } from "@inertiajs/react";
+import React, { ComponentProps } from "react";
 
-type Props = InertiaLinkProps;
+import { classNames } from "../utils/classNames";
 
-export const SecondaryButton = ({ children, ...props }: Props) => {
+type Props = {
+  orientation?: string;
+} & ComponentProps<"button">;
+
+export const SecondaryButton = ({ children, className, ...props }: Props) => {
```

Comment:
I would prefer to invert the naming (both in the filename as in the variable name).

```suggestion
export const ButtonSecondary = ({ children, className, ...props }: Props) => {
```